### DISCUSSION TITLE
NewPipe taking up 3GB

### DISCUSSION THREAD

**Discussion by aentrenus** [TeamNewPipe/NewPipe#10687] [Questions]
The NewPipe app on my phone is taking up almost 3GB of space, 2.5GB of which is a singular .tmp file in /files. I am kinda short on space rn, so I wanted to know, is this necessary?

**Comment by opusforlife2** [comment]
Unless you really care about your download history list in the app, you could just export your database, reinstall the app, and then import.

**Reply by aentrenus** [reply]
Thanks!

**Reply by TobiGr** [reply]
This looks like a paused or interrupted download to me. Might be interesting to look up what happens to paused and interupted (e.g. due to network connection errors) downloads
